### DISCUSSION TITLE
Config Bac-à-sable

### DISCUSSION THREAD

**Discussion by jodionrq** [MTESSDev/FRW#15] [Annonces]
Bonjour,

Nous avons installé Visual Studio Code, le Bac-à-sable MTESS, et l'extension yaml, pour permettre à une collègue de visualiser notre formulaire dans FRW. De notre côté tout fonctionne, mais du côté de notre collègue, il y a un message d'erreur lorsqu'elle appuie sur "ctrl+s" pour visionner le formulaire dans le Bac-à-sable (voir capture d'écran ci-dessous).

Avez-vous une solution?
Merci!

![MTESS-erreur](https://user-images.githubusercontent.com/124822406/219090397-1c674b77-0100-4864-a058-cbe98a29d74e.png)

**Comment by j0ber** [comment]
Bonjour,
Quelles valeurs sont dans la config de l'extension MTESS - Bac à sable pour votre collègue? Avez-vous la bonne url?
ex. 
![image](https://user-images.githubusercontent.com/26974817/219108474-ab4109b2-05c4-48c6-b552-9ced2974185f.png)

**Reply by jodionrq** [reply]
Bonjour,

Merci pour la réponse, c'est exactement ce qu'il manquait.
Bonne journée!
